### PR TITLE
NTR Helper Usage

### PR DESCRIPTION
**Pull request by OPBreloom**
# Guide Details

## I want to (select one)

- [ x ] Add a guide
- [ ] Modify an existing guide

## Games affected

- Gen 6/7

## Type of RNG

NTR Helper

## Language

ENG

## Notes

If needed, please make small notes here for extra details.

### CODE REVIEW

File: guides/tools/en/NTR Helper Usage.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# NTR Helper Usage
+
```
Comment (ShinySylveon04) [inline guides/tools/en/NTR Helper Usage.md]: Something I forgot that the guide needed was a short description in the beginning that is italicized.

Please insert a short description here about what the guide is for following this format:

>_description_

File: guides/tools/en/NTR Helper Usage.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## Using NTR
+
+### Enabling Debugger
+
+- If you are using an O2DS or O3DS you will need to use NTR Mode 3 for PCalc.
+- For instructions on how to set up and use NTR for use with PCalc follow [this guide](https://pokemonrng.com/guides/tools/en/How%20to%20Install%20PCalc/).
+- Additionally you will also need to enable debugger to be able to connect to NTR Helper.
+    - This is necessary for the console to stay connected to the internet after loading the game.
+   - Press `X + Y` to bring up the NTR menu then select "Enable Debugger".
```
Comment (Subject21J) [inline guides/tools/en/NTR Helper Usage.md]: Delete the 3 spaces and do a tab here for the two lines.

File: guides/tools/en/NTR Helper Usage.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+- Otherwise for N3DS or N2DSXL run NTR normally without Mode 3.
+    - The debugger is automatically enabled in NTR.
+
+### Finding IP Address
+
+- The IP address can be found on the Rosalina menu screen if Input Redirection is enabled. Input Redirection is not needed but it's a quick way to see the IP address. 
+   - To open the Rosalina menu screen press `Select + L + Down` on the 3DS and enable Input Redirection.
+   - If wanted you can disable Input Redirection after obtaining the IP address in the same way you enabled it.
+
+
```
Comment (Subject21J) [inline guides/tools/en/NTR Helper Usage.md]: Remove empty line.

File: guides/tools/en/NTR Helper Usage.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+4. Fill in IP address.
+    - See above for finding the console's IP address.
+
+5. Click "One Click" to connect in the RNG Helper window within 3DSRNGTool.
+
+### Reseeding
+
+The initial seed is generated once you enter the continue screen. This allows us to generate a new initial seed without resetting the game or console.
+
+
```
Comment (Subject21J) [inline guides/tools/en/NTR Helper Usage.md]: Remove empty line.

File: guides/tools/en/NTR Helper Usage.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+To reset the the initial seed:
+1. Press `B` to go back to the title screen from the continue screen.
+2. Press `A` to go back to the continue screen, at which point a new initial seed should be generated.
+- These steps can be done as many times as needed.
+
+NTR Helper will automatically input the initial seed into 3DSRNGTool each time you reseed as long as it is connected to the 3DS. 
+
+
```
Comment (Subject21J) [inline guides/tools/en/NTR Helper Usage.md]: Remove two empty lines.

File: guides/tools/en/NTR Helper Usage.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+    - This is necessary for the console to stay connected to the internet after loading the game.
+   - Press `X + Y` to bring up the NTR menu then select "Enable Debugger".
+   - Press `Select + L + Down` to bring up the Rosalina menu then select "Enable Debugger".
+- Otherwise for N3DS or N2DSXL run NTR normally without Mode 3.
+    - The debugger is automatically enabled in NTR.
+
+### Finding IP Address
+
+- The IP address can be found on the Rosalina menu screen if Input Redirection is enabled. Input Redirection is not needed but it's a quick way to see the IP address. 
+   - To open the Rosalina menu screen press `Select + L + Down` on the 3DS and enable Input Redirection.
```
Comment (Subject21J) [inline guides/tools/en/NTR Helper Usage.md]: change the wording here to: Open the Rosalina menu screen by pressing `Select + L + Down` on the 3DS. Under Miscellaneous options enable InputRedirection.

File: guides/tools/en/NTR Helper Usage.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+   - If wanted you can disable Input Redirection after obtaining the IP address in the same way you enabled it.
```
Comment (Subject21J) [inline guides/tools/en/NTR Helper Usage.md]: change the wording here to: InputRedirection can be turned off the same way it was enable if it's not needed.

File: guides/tools/en/NTR Helper Usage.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
+# NTR Helper Usage
+
+_How to Use NTR Helper with 3DSRNGTool_
+
+NTR Helper can be used to connect to a CFW 3DS for automatically grabbing information from PCalc, such as the initial seed or the TinyMT seeds. This reduces time from having to manually input the information, and 3DSRNGTool will automatically update the initial seed or TinyMT seeds when needed.
```
Comment (Subject21J) [inline guides/tools/en/NTR Helper Usage.md]: Somewhere here or in the next section, add a note that an internet connection is needed. Both the pc, that is running 3dsrngtool, and the 3ds needs to be connected to the same network.

File: guides/tools/en/NTR Helper Usage.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+- Additionally you will also need to enable debugger to be able to connect to NTR Helper.
+    - This is necessary for the console to stay connected to the internet after loading the game.
+   - Press `X + Y` to bring up the NTR menu then select "Enable Debugger".
+   - Press `Select + L + Down` to bring up the Rosalina menu then select "Enable Debugger".
+- Otherwise for N3DS or N2DSXL run NTR normally without Mode 3.
+    - The debugger is automatically enabled in NTR.
+
+### Finding IP Address
+
+- The IP address can be found on the Rosalina menu screen if Input Redirection is enabled. Input Redirection is not needed but it's a quick way to see the IP address.
```
Comment (Subject21J) [inline guides/tools/en/NTR Helper Usage.md]: Change Input Redirection to InputRedirection as that how the creator named it. InputRedirection is needed for the tid bot to work.

**Comment by zaksabeast** [comment]
Updated, merged, and squashed in e7da7ad8205b9671751345379a110ac466af683d.

**Pull request closed without merging**